docker/docker-compose.yml

```diff
@@ -33,9 +33,9 @@ services:
       - influxdb:/var/lib/influxdb/
       - "manager-config:/var/lib/chroma"
     healthcheck:
-      test: ["CMD-SHELL", "influx -host influxdb -execute 'show databases'"]
+      test: ["CMD-SHELL", "curl --fail influxdb:8086/ping?wait_for_leader=10s"]
```

Comment:
This can also just use `influx -host influxdb -execute 'exit'` instead.  It's what's used during setup to wait for influx to be online.

Reply:
Looks like that takes ~ 50% of a core:

![Screen Shot 2020-06-18 at 11 08 10 AM](https://user-images.githubusercontent.com/458717/85038012-166f8d00-b154-11ea-941e-2b392e053fd1.png)


Keep in mind this is being run every 5s.